FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Exceptions/SynapsePipelineRetriableException.cs

```diff
@@ -5,16 +5,16 @@
 
 using System;
 
-namespace Microsoft.Health.Fhir.Synapse.Core.Exceptions
+namespace Microsoft.Health.Fhir.Synapse.Common.Exceptions
 {
-    public class ExecuteTaskFailedException : Exception
+    public class SynapsePipelineRetriableException : Exception
```

Comment:
How do we handle/distinguish retriable and internal exceptions?

Reply:
Renamed `retriable` to `external`.
External means customer error and internal means services error,